types/hubot/index.d.ts

```diff
@@ -1,17 +1,63 @@
-// Type definitions for hubot 2.19
+// Type definitions for hubot 3.3
 // Project: https://github.com/hubotio/hubot
 // Definitions by: Dirk Gadsden <https://github.com/dirk>
 //                 Kees C. Bakker <https://github.com/KeesCBakker>
 //                 Emil Marklund <https://github.com/eeemil>
+//                 Jon Phenow <https://github.com/jphenow>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
-
-/// <reference types="node" />
+// TypeScript Version: 3.7
```

Comment:
Out of curiosity, which 3.7 features do these new types depend on?

Reply:
Oh when I started I didn't realize it had more meaningful implications. I set it to the version _I_ was on. I can drop it to the lowest common denominator.

Reply:
Was able to drop to 3.0 - needed to go that high for `unknown`